src/main/scala/polimerasa/QuickManualTester(removeLater).scala

```diff
@@ -0,0 +1,27 @@
+package polimerasa
+
+import polimerasa.alignments.algorithms.SequenceMatching
+
+
+@main
```

Comment:
Use ScalaTest instead of writing tests here

Reply:
Testy dodane, w końcówce algosa był błąd, został naprawiony